### PR TITLE
For issue #7720: ETP toggle accessibility updates

### PR DESCRIPTION
**Pull request by asavill**
### Pull Request checklist

- [x] **Tests**: Small UI tweak no additional tests
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not

**Before**
Screenshot_20200323-154921

old

**After**
Screenshot_20200323-144826
new

- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by asavill** [comment]
> No Taskcluster jobs started for this pull request
> 
> ```js
> The `allowPullRequests` configuration for this repository (in `.taskcluster.yml` on the
> default branch) does not allow starting tasks for this pull request.
> ```

Is this something I can solve on my end?

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1025,6 +1025,10 @@
     <string name="etp_panel_on">Protections are ON for this site</string>
     <!-- Enhanced Tracking Protection message that protection is currently off for this site -->
     <string name="etp_panel_off">Protections are OFF for this site</string>
+    <!-- Text to announce when Enhanced Tracking Protection switch is ON -->
```
Comment (asavill) [inline app/src/main/res/values/strings.xml]: What is the process for localising these strings? Is it via Pontoon?

**Comment by ekager** [comment]
Thank you! This looks good. We're currently having a CI issue where we can't run on contributor PRs so I've pulled your commits into https://github.com/mozilla-mobile/fenix/pull/9474 (you'll still get the commit credit!)

Yes the strings will be pulled out and sent to pontoon for localization as part of our l10n process

**Comment by ekager** [comment]
Closing for #9474

**Pull request closed without merging**